bindings/rust/evmc-declare-tests/src/lib.rs

```diff
@@ -0,0 +1,22 @@
+use evmc_vm::EvmcVm;
+use evmc_vm::ExecutionResult;
+use evmc_vm::ExecutionContext;
+#[macro_use]
+use evmc_declare::evmc_declare_vm;
+
+#[evmc_declare_vm(FooVm = "ewasm")]
```

Comment:
Is it possible to merge `evmc-declare-tests` into a test section in `evmc-declare`?

Reply:
No, you cannot test a procedural macro in the same crate :(
`proc_macro2` does have some facilities for this but I prefer to test via code expansion, etc.

Reply:
Ok, lets keep the extra crates then.